source/mrmeshpy/MRPythonPolyline.cpp

```diff
@@ -0,0 +1,102 @@
+#include "MRMesh/MRPython.h"
+#include "MRMesh/MRMeshFwd.h"
+#include "MRMesh/MRPolyline.h"
+#include "MRMesh/MRVector3.h"
+#include "MRMesh/MRVector2.h"
+#include "MRMesh/MRMesh.h"
+#include "MRMesh/MRAffineXf2.h"
+#include "MRMesh/MRAffineXf3.h"
+#include "MRMesh/MRBox.h"
+#include "MRMesh/MRAABBTreePolyline.h"
+#include "MRMesh/MR2DContoursTriangulation.h"
+#include "MRMesh/MRSymbolMesh.h"
+
+
+
+MR_ADD_PYTHON_CUSTOM_DEF( mrmeshpy, MeshSignedDistanceResult, []( pybind11::module_& m )
```

Comment:
Why MeshSignedDistanceResult and not polyline?

Reply:
fixed